### PR TITLE
Use torch.nn.init in SqueezeNet models

### PR DESCRIPTION
**Pull request by Maratyszcza**
I verified that SqueezeNet 1.1 converges to about the same accuracy after the change

### CODE REVIEW

**Comment by fmassa** [comment]
Thanks!
